### PR TITLE
remove redundant unittest.mock imports

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: src/pytest_mock/plugin.py [hunk 1]
```diff
@@ -87,7 +84,7 @@ def spy(self, obj: object, name: str) -> unittest.mock.MagicMock:
 
         :param object obj: An object.
         :param unicode name: A method in object.
-        :rtype: mock.MagicMock
+        :rtype: unittest.mock.MagicMock
```
Comment (graingert) [inline src/pytest_mock/plugin.py:87]: do we care about `:rtype:`s here? can we use sphinx-autodoc-typehints instead?

Comment (nicoddemus) [reply src/pytest_mock/plugin.py:87]: Hey @graingert! 

IMHO we can drop all types from the docs and rely solely on the type annotations.

About sphinx, we don't really have sphinx docs at the moment, so I don't think it matters.

Want to tackle this in this PR?

**Comment by nicoddemus** [comment]
I don't think this applies anymore given we use `unittest.mock.MagicMock` for annotations.

Feel free to reopen if I'm wrong however!

**Comment by graingert** [comment]
unittest.mock is imported 3 times, we only need it once

File: src/pytest_mock/plugin.py (hunk 1 above)
Comment (graingert) [reply src/pytest_mock/plugin.py:87]: I think worth defering to another pr

**Comment by nicoddemus** [comment]
Ahh right, my bad! Sorry about that.

**Pull request merged**